vm/vm.go

```diff
@@ -15,15 +15,15 @@ import (
 )
 
 type VmOptions struct {
```

Comment:
```suggestion
type Options struct {
```

Unrelated to the PR changes, but can we handle this Lint Warning? Thank You.

Reply:
Sure @daemon1024 I will handle this.